### PR TITLE
UPSE-293: updates to support Java 11

### PR DESCRIPTION
**Pull request by groybal**
Added JAXB dependencies for Java 11.
Downgraded Logback version to one that runs with Java 8 (tricky: runtime error only -- no compile error).

### CODE REVIEW

File: resource-server-webapp/pom.xml [hunk 1]
```diff
@@ -158,7 +170,7 @@
 				<executions>
 					<execution>
 						<id>sign-artifacts</id>
-						<phase>verify</phase>
+						<phase>deploy</phase>
```
Comment (ChristianMurphy) [inline resource-server-webapp/pom.xml:173]: Is this related to the jaxb/logback changes?
Is this needed for Java 11?

**Review by ChristianMurphy — COMMENTED** [review]
Thanks @groybal!
Logback and Jaxb changes look good!
Have a few questions on some of the other changes in here.

File: resource-server-webapp/src/main/resources/logback.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
 <!--    http://www.qos.ch/shop/products/professionalSupport         -->
 <!--                                                                -->
 <configuration scan="true" scanPeriod="30 seconds">
-  <contextName>ResourceServingWebapp</contextName>
+  <contextName>resource-server</contextName>
```
[the remark this answers is not in the document]
Comment (groybal) [reply resource-server-webapp/src/main/resources/logback.xml:30]: We have two resource server webapps running, an older one (ResourceServingWebapp version 1.0.48) and a newer one (resource-server version 1.4.x) but they both log to the same log file so it was confusing to debug issues.  I can remove this change if we still want both webapps to log to same file.

File: resource-server-webapp/pom.xml (hunk 1 above)
Comment (groybal) [reply resource-server-webapp/pom.xml:173]: If I'm remembering correctly, with 'verify', the build was failing with Java 11.  I changed it to 'deploy' to resolve because I figured we don't need to sign artifacts anyways unless we're doing a deploy. Now that I think about it, if we run a deploy the same error will probably still occur :(

Comment (groybal) [reply resource-server-webapp/pom.xml:173]: Okay so I retested using 'verify' and I get build failure with both Java 8 and Java 11.  Here's the error, anyone have insights:
```[DEBUG] Generating signature for /home/groybal/projects/unicon/uportal/forks/resource-server/resource-server-webapp/target/ResourceServingWebapp.war
[DEBUG] 2.2.19
[GNUPG:] INV_SGNR 9
[GNUPG:] FAILURE sign 17
gpg: no default secret key: No secret key
gpg: signing failed: No secret key```

Comment (ChristianMurphy) [reply resource-server-webapp/pom.xml:173]: I'd check

```bash
gpg --list-keys
```
to see if there is a key already setup on your system

If there isn't try running

```bash
gpg --generate-key
```

Comment (groybal) [reply resource-server-webapp/pom.xml:173]: Yep, that fixed it (generating key).  Still should use 'deploy' I think but let me know if I should change back to 'verify'.

Comment (ChristianMurphy) [reply resource-server-webapp/pom.xml:173]: I'm good with it staying in deploy as well.
Though open to differing opinions.

**Review by loulou2u — APPROVED** [review]
This works great! I had to revise the logback version but it's working. Thanks
